app/views/home/contributors.md

```diff
@@ -0,0 +1,69 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that might be looking for ways to use those skills on an open source project.
+
+> Heading: A call to action. What is the one thing you want me to do? (3-6 words)
+
+# Make a contribution
+
+> Lead: Ok, you have my attention. Tell me more. Why should I care and what should I do about it? (~20ish words)
+
+Open source software is made by people just like you. This Friday, join the community and make one contribution.
+
+---
+
+> Why: this section should be the elevator pitch for why this person should want to get involved (5-10 words)
+
+## Get the most out of open source
```

Comment:
this is something I wrote in medium, but I think for example for myself (learning dev) will be the most attractive aspect of participating and it can also resonate with experienced career devs. I think...
Working on open source projects can be incredibly rewarding. You will be helping your favorite cause solve problems, or maybe working on a new project within your company different from your responsibilities. Doing this while building your public code base of contributions one that can show your colleagues and potential employers your ability to work collaboratively and your eagerness to learn.